app/workers/github/create_issue_job.rb

```diff
@@ -30,8 +30,12 @@ def within_limit
     def perform(feedback)
       feedback = Feedback.new(feedback)
       THROTTLE.within_limit do
-        create_response = Github::GithubService.create_issue(feedback)
-        FeedbackSubmissionMailer.build(feedback, create_response&.html_url, create_response&.number).deliver_now
+        # create_response = Github::GithubService.create_issue(feedback)
+        FeedbackSubmissionMailer.build(
+          feedback,
+          'Automatic Github issue creation skipped.',
```

Comment:
Does the github link parameter get used for anything, seems like no but I may be missing something. Is skipping issue creation permanent? If yes maybe we don't need it as part of the mailer?

Reply:
> Does the github link parameter get used for anything

[It gets used in the template that creates the body of the email](https://github.com/department-of-veterans-affairs/vets-api/blob/master/app/mailers/views/feedback_report.erb#L18).

> Is skipping issue creation permanent?

[Maybe.  Maybe not.](https://github.com/department-of-veterans-affairs/vets.gov-team/issues/8277#issuecomment-358979244)